lib/page_model.dart

```diff
@@ -165,7 +165,7 @@ class PageModel {
   WidgetModel buildRootModel(YamlMap viewMap, Map<String, dynamic>? customViewDefinitions) {
     WidgetModel? rootModel = getRootModel(viewMap, customViewDefinitions!);
     if (rootModel != null) {
-      if (![Column.type, Row.type, Flex.type, EnsembleStack.type, AppScroller.type].contains(rootModel.type)) {
+      if (![Column.type, Row.type, Flex.type, EnsembleStack.type, AppScroller.type, ListView.type].contains(rootModel.type)) {
```

Comment:
remove this. ListView doesn't need to be the root widget. The rootwidget is meant as a wrapper to include other widgets.